eras/shelley/impl/src/Cardano/Ledger/Shelley/Core.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Cardano.Ledger.Shelley.Core
+  ( ShelleyEraTxBody (..),
+    Wdrl (..),
+    module Cardano.Ledger.Core,
+  )
+where
+
+import Cardano.Binary (FromCBOR (..), ToCBOR (..))
+import Cardano.Ledger.Address (RewardAcnt (..))
+import Cardano.Ledger.Coin (Coin (..))
+import Cardano.Ledger.Core (Era (..), EraTxBody (..), ExactEra)
```

Comment:
The idea was to re-export the whole of `Core`, so in all other modules we can import `import Cardano.Ledger.Shelley.Core` and not bother with `import Cardano.Ledger.Core`. For this we need to import that module as a whole
```suggestion
import Cardano.Ledger.Core
```